src/components/pages/WikiNotePage.tsx

```diff
@@ -141,6 +141,8 @@ const WikiNotePage = ({ storage }: WikiNotePageProps) => {
         {note != null && generalStatus.showingNoteContextMenu && (
           <NoteContextView storage={storage} note={note} />
         )}
+        <div id='flowchart-export' style={{ display: 'none' }} />
```

Comment:
A bit of hack to render charts for exports inside page since otherwise they cannot be rendered without proper loading of `BrowserWindow`

Reply:
Refactored, now added to body dynamically